lib/node4Migration.js

```diff
@@ -0,0 +1,161 @@
+var Superagent = require('superagent');
+var Async = require('async');
+var Chalk = require('chalk');
+var Sandbox = require('sandboxjs');
+
+exports.isNode4Webtask = function isNode4Webtask(webtask) {
+    return webtask && webtask.sandbox 
+        && exports.isNode4Profile(webtask.sandbox);
+};
+
+exports.isNode4Profile = function isNode4Profile(profile) {
+    return profile && profile.url 
+        && profile.url.match(/^https:\/\/webtask.it.auth0.com\/?$/i);
+};
+
+exports.node8BaseUrl = 'https://sandbox.auth0-extend.com';
+
+exports.migrate = function migrate(options, cb) {
+
+    tryMigrate(10);
+
+    function tryMigrate(remainingAttempts) {
+        if (remainingAttempts === 0) 
+            return cb(new Error('Webtask did not finish migrating within allotted time. Try again.'));
+        Superagent
+            .post(`https://webtask-migrate.sandbox.auth0-extend.com/${ process.env.WT_MIGRATE_TEST == 1 ? 'test' : 'prod'}`)
+            .send({ 
+                containerName: options.containerName, 
+                webtaskName: options.webtaskName,
+                replaceIfExists: true,
```

Comment:
Shouldn't this be conditional based on the options?

Reply:
If the _actual_ migration is predicated on whether we run the `test` or `prod` webtask, then we should drop this flag.

Reply:
I am implementing the "override" logic explicitly in `migrate.js`, so if the code goes down here is always to override.